linear_algebra/src/conjugate_gradient.py

```diff
@@ -0,0 +1,172 @@
+import numpy as np
```

Comment:
Please add a URL that can help the reader learn about the problems that your algorithm is trying to solve...
https://en.wikipedia.org/wiki/Conjugate_gradient_method
https://en.wikipedia.org/wiki/Definite_symmetric_matrix

Reply:
Added :)